### PR TITLE
Added support for arm builds via qemu.

### LINKED ISSUE #218: Stable version 4.8.5 from january not in latest branch
Hi,
is there a reason why version 4.8.5 is not included in latest? The version there is 4.8.4 and i'm wondering why it is like that since january, where 4.8.5 was released.

### LINKED ISSUE #248: Please support arm
arm
arm
arm

### PR DESCRIPTION
**Pull request by TrAnn3l**
Should resolve #248 
Used [https://www.trion.de/news/2019/10/14/docker-multi-arch-dockerhub.html](https://www.trion.de/news/2019/10/14/docker-multi-arch-dockerhub.html) and [https://github.com/ckulka/docker-multi-arch-example](https://github.com/ckulka/docker-multi-arch-example) as example.
One should just need to add the new Dockerfiles to the automated build configuration in Dockerhub and the rest should work.

### CODE REVIEW

**Comment by williamdes** [comment]
@J0WI LGTM but what do you think about this PR?

**Comment by williamdes** [comment]
Thank you @TrAnn3l!

**Comment by J0WI** [comment]
I still hope that @yosifkit or @tianon will review https://github.com/docker-library/official-images/pull/7288 soon :confused:

**Comment by williamdes** [comment]
> I still hope that @yosifkit or @tianon will review [docker-library/official-images#7288](https://github.com/docker-library/official-images/pull/7288) soon

Me too, I do not understand what blocs the review now, maybe they just have missed the notification
Maybe I should rebase 🤔

**Comment by williamdes** [comment]
Could you please add some documentation on the README @TrAnn3l ?

**Comment by TrAnn3l** [comment]
Yes, after i finished testing.
I created my own docker [repo](https://hub.docker.com/r/trann3l/phpmyadmin) and now i'm fixing small bugs. I will post my configuration when everything works.

File: fpm-alpine/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,6 @@
-FROM php:7.4-fpm-alpine
+FROM amd64/php:7.4-fpm-alpine
+
+# Add QEMU
```
Comment (williamdes) [inline fpm-alpine/Dockerfile:3]: Can be removed?

File: fpm/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,6 @@
-FROM php:7.4-fpm
+FROM amd64/php:7.4-fpm
+
+# Add QEMU
```
Comment (williamdes) [inline fpm/Dockerfile:3]: Same here :)

**Comment by TrAnn3l** [comment]
Ok,  i am finished. I tried to make it as simple and as vanilla as possible. I also tried to keep the workflow, so one only needs to change the templates and run update.sh. In addition you should not need to change the current Docker Hub config.

You can see the working result here: [https://hub.docker.com/r/trann3l/phpmyadmin](https://hub.docker.com/r/trann3l/phpmyadmin)
This is my Docker Hub config:
grafik

If you still have any questions, feel free to ask 😃

**Comment by williamdes** [comment]
Thanks you, I appreciate the initiative to keep things vanilla
@ibennetch could you give your thoughts on this PR?

**Comment by J0WI** [comment]
This is obsolete with https://hub.docker.com/_/phpmyadmin

**Comment by TrAnn3l** [comment]
Closing because of https://hub.docker.com/_/phpmyadmin.

**Pull request closed without merging**